client/README.md

```diff
@@ -66,9 +66,8 @@ const (
 
 func main() {
 	// Make client
-	u, _ := url.Parse("http://localhost:8086")
-	c := client.NewClient(client.Config{
-		URL: u,
+	c := client.NewHTTPClient(client.HTTPConfig{
```

Comment:
This would be a breaking change to the client, and we can't do that unfortunately.

The v2 client is where all new client development is taking place.